lib/galaxy/datatypes/sniff.py

```diff
@@ -253,75 +254,75 @@ def is_column_based( fname, sep='\t', skip=0, is_multi_byte=False ):
     return True
 
 
-def guess_ext( fname, sniff_order=None, is_multi_byte=False ):
+def guess_ext( fname, sniff_order, is_multi_byte=False ):
```

Comment:
`tools/data_source/upload.py` contains this line:
```
ext = sniff.guess_ext( dataset.path, is_multi_byte=True )
```
which will break after this change.